app/bundles/CoreBundle/Tests/unit/Helper/InputHelperTest.php

```diff
@@ -56,4 +56,16 @@ public function testHtmlFilter()
             $this->assertEquals($expected, $actual);
         }
     }
+
+    /**
+     * @testdox The email returns value without double period
+     *
+     * @covers \Mautic\CoreBundle\Helper\InputHelper::email
+     */
+    public function testEmailFilterRemovesDoublePeriods()
```

Comment:
Maybe add the to `trim`, which was added too?

Reply:
Done